DistributedResponseCachingMiddleware/Utility/CustomResponseCachingPolicyProviderExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.ResponseCaching;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace ExternalNetcoreExtensions.Utility
```

Comment:
All extensions to IServiceCollection should be in the ` Microsoft.Extensions.DependencyInjection` namespace. This makes them more discoverable with intellisence.